src/main/scala/nl.knaw.dans.easy.deposit/docs/AgreementsXml.scala

```diff
@@ -17,15 +17,16 @@ package nl.knaw.dans.easy.deposit.docs
 
 import org.joda.time.DateTime
 
-import scala.util.Try
+import scala.util.{ Failure, Try }
 import scala.xml.Elem
 
 object AgreementsXml extends SchemedXml {
   override val schemaNameSpace = "http://easy.dans.knaw.nl/schemas/bag/metadata/agreements/"
-  override val schemaLocation = "https://easy.dans.knaw.nl/schemas/bag/metadata/agreements/2019/01/agreements.xsd"
+  override val schemaLocation = "https://easy.dans.knaw.nl/schemas/bag/metadata/agreements/2019/09/agreements.xsd"
 
-  def apply(userId: String, dateSubmitted: DateTime, dm: DatasetMetadata, fullname: String): Try[Elem] = {
-    for {
+  def apply(dateSubmitted: DateTime, dm: DatasetMetadata, userProperties: UserInfo): Try[Elem] = {
```

Comment:
How about 
```suggestion
  def apply(dateSubmitted: DateTime, dm: DatasetMetadata, user: UserInfo): Try[Elem] = {
```

change the reference(s) below yourself